src/quickgui/plugin/qgsquickpositionmarker.qml

```diff
@@ -0,0 +1,113 @@
+/***************************************************************************
+  qgsquickpositionmarker.qml
+  --------------------------------------
+  Date                 : Dec 2017
+  Copyright            : (C) 2017 by Peter Petrik
+  Email                : zilolv at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.3
+import QtQuick.Controls 2.2
+import QtQml 2.2
+import QtGraphicalEffects 1.0
+import QgsQuick 0.1 as QgsQuick
+
+/**
+ * \brief A marker refers to the current gps position according gps source is available. Along with longitude, latitude and radius.
```

Comment:
brief is just "brief" -- one short sentence. also it may not be GPS.. .

I would say something like

Graphical representation of physical location on the map

Source position and accuracy taken from PositionKit is drawn as a marker on the map. Marker is grayed out when position is not available. When PositionKit support reading of the accuracy, the circle is drawn around the marker. PositionKit must be connected, for example GPS position source from QgsQuickPositionKit.